lib/ansible/modules/identity/1password_facts.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Ryan Conway (@rylon)
+# (c) 2018, Scott Buchanan <sbuchanan@ri.pn> (onepassword.py used as starting point)
+# (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+# Ansible boilerplate: https://docs.ansible.com/ansible/2.6/dev_guide/testing/sanity/boilerplate.html
```

Comment:
This line is not needed.

Reply:
Ah yes, that was for me, I will remove it.